python-packages/securesync/templates/securesync/register_public_key_client.html

```diff
@@ -99,53 +74,46 @@
 
 
     {% if unregistered %}
-    <div class="chunk-container">
-        <h2>{% trans "What is device registration?" %}</h2>
-        <p class="normal-size" style="padding: 0px 40px 0px 40px;">
+    <div class="row">
+        <h2 class="col-md-12">{% trans "What is device registration?" %}</h2>
+        <p class="col-md-10" style="margin-left:40px">
```

Comment:
Hi @rtibbles and @jtamiace thanks for the review.

I removed the `inline css` and make sure that the class names and selectors are unique. 
I used a prefix ` reg-dev` for the selectors and class names for this html.